app/src/main/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIClient.java

```diff
@@ -353,7 +356,17 @@ public void postImg(final Context context, final ProductImage image) {
         lt.setTextColor(context.getResources().getColor(R.color.white));
         lt.show();
 
-        apiService.saveImage(image.getCode(), image.getField(), image.getImguploadFront(), image.getImguploadIngredients(), image.getImguploadNutrition())
+        String lang = Locale.getDefault().getLanguage();
+
+        HashMap<String, RequestBody> imgMap = new HashMap<>();
```

Comment:
use a generic interface for declaration like `Map<String, RequestBody> imgMap = new HashMap<>();`